index.html

```diff
@@ -1294,6 +1449,45 @@ <h2><dfn>CaptureController</dfn>
               </li>
             </ol>
           </dd>
+
+
+          <dt><dfn>setDisplaySurfaceChangeCallback</dfn>
+          </dt>
+
+
+          <dd>
+            <p>Run the following steps:</p>
+
+
+            <ol>
+              <li>
+                <p>Let <var>callback</var> be the method's first argument.</p>
+              </li>
+
+
+              <li>
+                <p>If [=this=].{{CaptureController/[[IsBound]]}} is
```

Comment:
It seems restrictive to disallow setting the callback more dynamically.
Why should we add this restriction?